### DISCUSSION TITLE
All entities are "unavaliable"

### DISCUSSION THREAD

**Discussion by Dumonster** [alandtse/alexa_media_player#1580] [Q&A] [answered]
I've recently setup this integration and I can't get it to work.

Everything sets up properly, the login is done but in the end all 7 entities show up as "Unavailable". I've even tried providing the 2FA key and it still doesn't work.

I'm using Nabu Casa, if it's relevant.

I've tried removing the integration a bunch of times with different settings but in the end it turns out the same.

![Untitled](https://user-images.githubusercontent.com/8132685/165537273-41e9f5af-890c-4f60-8b9f-4aa244ceecf0.png)

I'm all out of ideas. Can somebody help me?

**Comment by Dumonster** [comment, marked as answer]
Nevermind. I was trying to connect to Amazon.es since that's the one I use daily but I should have been using Amazon.com.

**Reply by phlaange** [reply]
I did the same thing by entering amazon.com.au. Thanks for posting a follow-up.
